### DISCUSSION TITLE
How to add RMD r markdown file to website?

### DISCUSSION THREAD

**Discussion by seanchien4** [daattali/beautiful-jekyll#1060] [Q&A] [answered]
I created a RMD file for R tutorial. How can I add into the website. It also includes some code and plots. It can be just plain, no need to be fancy. I just want the address would be user.github.io/filename/ 

Thank you!

**Comment by daattali** [comment, marked as answer]
You'll have to render the Rmd to an HTML page yourself. If you just place the html page in the root, it'll be accessible. For example if you have a file called `test.html` then it'll be accessible at `user.github.io/test`

If you want to display the html page as-is, then that's all you need to do.

If you want the page to inherit beautifuljekyll's templat, then remember to add the frontend YAML (see the README for more info)

```
---
layout: page
title: 'Page title'
---
```

**Reply by seanchien4** [reply]
Thanks! it works.
